lib/modules/header-widgets/views/widget.html

```diff
@@ -27,14 +27,14 @@ <h1>{{data.widget.title}}</h1>
                     </div>
                 </div>
                 <div class="col-sm-3 col-xs-12 header-buttons">
-                    <a href="/plannen" class="margin-bottom-xs scroll-link block no-decoration">
+                    <a href="{{ data.widget._ideaOverviewPage._url if (data.widget._ideaOverviewPage and data.widget._ideaOverviewPage._url) else '/plannen' }}" class="margin-bottom-xs scroll-link block no-decoration">
```

Comment:
Maybe it's nicer to implement a helper method something like this:
`data.widget.getIdeaOverviewPageUrl()`